Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -567,7 +567,7 @@ void ControllerConfigDiag::SaveWiimoteSource()
 
   for (unsigned int i = 0; i < MAX_WIIMOTES; ++i)
   {
-    std::string secname("Wiimote");
+    std::string secname("Wii Remote");
```

Comment:
Yes. Let's only change the user-visible strings, not the strings that are used in INIs.